### DISCUSSION TITLE
Perl color highlight

### DISCUSSION THREAD

**Discussion by SamuelB-ACA** [outline/outline#2634] [Self Hosting] [answered]
Hi,
Is it possible to enable color highlighting for perl code

**Comment by tommoor** [comment, marked as answer]
Hi Sam, just added – it will be deployed later today to cloud hosted and available in the next release of OSS edition.

**Reply by SamuelB-ACA** [reply]
Awesome, tried it on cloud hosted, working perfectly. Thanks !!
